dist/aframe-teleport-controls.js

```diff
@@ -52,12 +52,17 @@
 	  throw new Error('Component attempted to register before AFRAME was available.');
 	}
 
+	var buttons = ['trackpad', 'trigger', 'grip', 'menu', 'thumbstick', 'A', 'B', 'X', 'Y', 'surface', 'point', 'pointing', 'pistol', 'thumb'];
```

Comment:
I wouldn't overcomplicate it adding these `buttons` and `events`. It will let many invalid combinations of both: `trackpadclose, thumbopen, pointtouchstart, ...`
I'll just leave the standard buttons for the controllers and use `up` and `down` as start/end. 
Also we're mixing here events from the `hand-controls` like `pointing`,  `pointing` with buttons events directly from the `tracked-controls` which is confusing.